bigtable/google/cloud/bigtable/cluster.py

```diff
@@ -172,3 +194,16 @@ def delete(self):
         """
         client = self._instance._client
         client.instance_admin_client.delete_cluster(self.name)
+
+    def create_pb_request(self):
```

Comment:
Should this method be private?  ISTM that the only caller is going to be `Instance.create`.

Reply:
Yes it should be private. Done